keadm/cmd/keadm/app/cmd/util/exec.go

```diff
@@ -64,14 +72,28 @@ func (cmd Command) GetCommand() string {
 
 func (cmd Command) GetStdOut() string {
 	if len(cmd.StdOut) != 0 {
-		return strings.TrimSuffix(string(cmd.StdOut), "\n")
+		return strings.TrimSuffix(ConvertByte2String(cmd.StdOut, "GB18030"), "\n")
 	}
 	return ""
 }
 
 func (cmd Command) GetStdErr() string {
 	if len(cmd.StdErr) != 0 {
-		return strings.TrimSuffix(string(cmd.StdErr), "\n")
+		return strings.TrimSuffix(ConvertByte2String(cmd.StdErr, "GB18030"), "\n")
 	}
 	return ""
 }
+
+func ConvertByte2String(byte []byte, charset string) string {
```

Comment:
It seems the input charset value is always `GB18030`? Does this affect the previous function for it used string(byte) ?

Reply:
I have split exec.go into exec_windows and exec_others, and this will make 'ConvertByte2String' only avaible in windows.
btw, i knew few about the character encoding in windows and found few information about "how to deal with windows 
 stdout containing Chinese characters", so keep a charset parameter in case we need it in the future(GB18030 work fine with me in my environment)